process/graduation_criteria.md

```diff
@@ -0,0 +1,118 @@
+  - Must have 
+    - Gated PRs with unit tests
+    - MSFT code of conduct
+    - LCA bot enabled
+
+## Incubating Stage
+An Incubating project is one that has met all of the requirements of the Sandbox stage and is 
+showing accelerating adoption.
+
+### Entry Requirements
+- Minimum of 3 customers using the project in a production or pre-production environment
+- Minimum of 2 maintainers
+- Full due diligence of the TOC
+  - A great example is the [CNCF Due Diligence Guidelines](https://github.com/cncf/toc/blob/fc161c13a86ba022277a906a10ff51a568406f7c/process/due-diligence-guidelines.md)
+  - TODO: create our own DD guidelines based on the CNCF DD guidelines
+- Demonstrate a substantial ongoing flow of commits and merged contributions
+- Have a healthy number of issues and pull requests from external contributors
+- Set a clear, documented versioning scheme
+- Show significant progress toward achieving [Core Infrastructure Initiative Best Practices Badge](https://bestpractices.coreinfrastructure.org/)
+  - Must have 
+    - Unit tests and integration tests covering a majority of the codebase
+    - Linting
+    - Static analysis (where appropriate)
+    - An initial security review
+    - Contributing guide
+    - Issue and PR templates
+    - Public project board with prioritization
+    - Milestones for releases
+- Telemetry for errors and general usage
+  - If not possible, provide reasoning for why
+
+## Graduation Stage
+Graduation Stage project is one that has met all of the requirements of the Incubation stage and has
+reached a level of maturity and polish which will delight customers.
+
+### Entry Requirements
+- Minimum of 30 customers using the project in production
+- Minimum of 4 maintainers (at least one from outside AzCU)
+- Semester project road map published and updated
+- Full due diligence of the TOC with an eye to any misalignment
+- Have achieved and maintained a [Core Infrastructure Initiative Best Practices Badge](https://bestpractices.coreinfrastructure.org/)
+  - Must have
+    - Load tests and performance tests
+  - Bonus for achieving Silver or higher
+- Full security review and threat analysis with the Azure Green / Red teams
+- Explicitly define a project governance and committer process. This preferably is laid out in a 
+  GOVERNANCE.md file and references an OWNERS.md file showing the current and emeritus committers.
+- Have a public list of project adopters for at least the primary repo (e.g., ADOPTERS.md or logos 
+  on the project website). For a specification, have a list of adopters for the implementation(s) 
+  of the spec.
+- SLA / SLO guidance
+- Support with ICM on-call rotations
+- Telemetry and Metrics for errors and general usage
+  - Define a set of KPIs with a dashboard
+  - Setup alerting for anomalous value ranges
+  - If not possible, provide reasoning for why
+
+## Donation Stage
```

Comment:
Given that this is taken from the CNCF graduation criteria, does this also apply to AzCU? It feels very specific to project donation to CNCF and less so to help us gate quality for our projects. WDYT?

Reply:
Donation is not a stage for CNCF. In fact, the project and IP is donated upon entrance into CNCF. This was intended to be more aspirational than about maturity. It was to guide us to aspire to making a project that out grows MSFT and Azure to become a foundational part of the OSS ecosystem so much so that the project moves into a more neutral governance model to better serve the ecosystem.

Reply:
are there ever reasons or times where donation to a foundation would happen ahead of project graduation from this ladder?

Reply:
Great points, @sarahnovotny and @ritazh. I've updated the document to state a project can be donated at any stage along the way. I'll also update the workflow.